app/views/external_users/admin/external_users/index.html.haml

```diff
@@ -40,7 +40,8 @@
         = govuk_table_td('data-label': t('.actions')) do
           - if advocate.active? && advocate.enabled?
             .app-link-group
-              = govuk_link_to t('.edit_html', context: advocate.name), edit_external_users_admin_external_user_path(advocate)
-              = govuk_link_to t('.delete_html', context: advocate.name), external_users_admin_external_user_path(advocate), method: :delete, data: { confirm: t('.confirmation') }
-          - else
+            = render_edit_delete_links(advocate)
```

Comment:
This may be a matter of personal preference so please ask the opinion of others. I would find this easier to follow if `render_edit_delete_links` were replace by a partial instead. Also, this helper (or partial. if you decide to do that) already has a `.app-link-group` div so the previous line (and the similar line below) are not required.